migrations/add-filter.js

```diff
@@ -0,0 +1,20 @@
+module.exports = {
+  async up(db) {
+    let Filter = require("../node_modules/bad-words");
+    const filter = new Filter({emptyList: true});
+    filter.addWords("covid-19","coronavirus", "pandemic","masks","mask")
+    const articles = await db.collection('articles').find({}).toArray();
```

Comment:
can we be consistent about double vs single quotes? is there a linter set in place for this repo?